lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -238,6 +238,8 @@ def store(payment, options = {})
 
               if options[:set_default] and r.success? and r.params['id'].present?
                 post[:default_card] = r.params['id']
+                post[:invoice_settings] = {}
+                post[:invoice_settings][:default_payment_method] = r.params['id']
```

Comment:
why just not? 🙂 
```
post[:invoice_settings] = { default_payment_method: r.params['id'] }
```

Reply:
I just followed the style used few lines above 🤔 
https://github.com/chargify/active_merchant/blob/16b842d15ccfefecec975b37b47b7ffb11e03269/lib/active_merchant/billing/gateways/stripe.rb#L232-L233